xdsl/frontend/code_generation.py

```diff
@@ -160,13 +165,18 @@ def visit_Compare(self, node: ast.Compare):
             "Lt": "slt",
             "LtE": "sle",
             "NotEq": "ne",
+            "In": "contains"  # Note: Not a real xDSL mnemonic!
```

Comment:
Having this here seems hacky @AlexanderViand? Why not split over standard comparisons and "is"/"is not"?

Reply:
Good point! "is"/"is not" aren't handled anyway, but I pulled out the handling of "in" into its own little case since I've realized the current code would actually fail if a frontend type tried to provide a "__contains__" overload/corresponding op since it'd try to pass a mnemonic, which really doesn't make sense in that case.